state-chain/runtime/src/runtime_apis.rs

```diff
@@ -166,6 +172,32 @@ pub enum EventFilter {
 	SystemOnly,
 }
 
+#[derive(Serialize, Deserialize, Encode, Decode, Eq, PartialEq, TypeInfo, Debug)]
+pub struct SafeModeStatuses {
+	pub swapping: pallet_cf_swapping::PalletSafeMode,
+	pub emissions: pallet_cf_emissions::PalletSafeMode,
```

Comment:
This struct looks equivalent to safe_mode::RuntimeSafeMode. 

If so, better to re-use that definition rather than create another one. 

(we probably just have to add Serialize/Deserialize in the impl_safe_mode macro definition).